apps/project/serializers.py

```diff
@@ -326,6 +360,33 @@ def update(self, instance, validated_data):
         framework = validated_data.get('analysis_framework')
         user = self.context['request'].user
 
+        organizations = self.context['request'].data.get('organizations')
+        project_organizations = ProjectOrganization.objects.filter(project=instance)
+
+        if organizations:
+            ids = [x.id for x in project_organizations]
+            request_ids = [x.get('id') for x in organizations]
+            to_remove = list(set(ids) - set(request_ids))
+
+            for i in to_remove:
+                ProjectOrganization.objects.get(id=i).delete()
+
```

Comment:
Some api tests for `update` scenario with different combinations of existing project-organizations and new project-organizations are missing.

Reply:
Added